visualization/package.json

```diff
@@ -39,6 +39,7 @@
 	},
 	"main": "dist/webpack/index.html",
 	"build": {
+		"nwVersion": "0.65.1",
```

Comment:
Note: I think at some point we removed it because it was causing some problem with updates. Renovate can't detect this line, so please make sure every time nw is updated to update this line manually!

Reply:
I assume that renovate is also able to take care of this fixed version. We should have a look on that later, because most likely this will be forgotten to be updated in the future.

Reply:
I added a regex to renovate.json to automatically check this for updates. I don't know how to test this, however. Any ideas?

Reply:
I have no idea how to test it, so maybe just wait and see what happens in the next update.